wrappers/javascript/package.json

```diff
@@ -55,7 +55,10 @@
     "tsconfig-paths": "4.1.0",
     "typescript": "4.5.5"
   },
+  "resolutions": {
+    "ref-napi": "npm:@2060.io/ref-napi"
```

Comment:
Can't we add this as a direct dependency for all the wrappers and not require the resolution anymore?

Reply:
I'm not sure if that would work because it is also an indirect dependency: `ffi-napi` requires it. In such case maybe we can also patch it to depend on the patched `ref-napi` as well.

Reply:
@TimoGlastra what about these latest changes? I needed some tweaks for types, but at the end seems to work fine. I think it's worth to put some effort on this considering the several problems users had in setting up these components for NodeJS.